tests/test_tileserver.py

```diff
@@ -1,21 +1,37 @@
 """Tests for tileserver."""
+import json
 import pathlib
-from pathlib import Path
+import urllib
+from pathlib import Path, PureWindowsPath
 from typing import List, Union
 
+import joblib
 import numpy as np
 import pytest
+from matplotlib import colormaps
 from shapely.geometry import LineString, Polygon
 from shapely.geometry.point import Point
 
 from tests.test_annotation_stores import cell_polygon
+from tests.test_utils import make_simple_dat
 from tiatoolbox.annotation.storage import Annotation, AnnotationStore, SQLiteStore
 from tiatoolbox.cli.common import cli_name
-from tiatoolbox.utils.misc import imwrite
+from tiatoolbox.utils.misc import imread, imwrite
```

Comment:
You can merge #606 and import as below
```suggestion
from tiatoolbox.utils import imread, imwrite
```